integration/grpc-web-no-streaming-observable/example.ts

```diff
@@ -436,3 +437,10 @@ export type Exact<P, I extends P> = P extends Builtin
 function isSet(value: any): boolean {
   return value !== null && value !== undefined;
 }
+
+export class GrpcWebError extends Error {
+  constructor(message: string, public code: grpc.Code, public metadata: grpc.Metadata) {
+    super(message);
+    Object.setPrototypeOf(this, GrpcWebError.prototype);
```

Comment:
@zakhenry what's the `setPrototypeOf` for? I've not seen it in constructors before.

Reply:
@stephenh it is a recommendation for extending built-in classes to ensure `instanceof` checks still work. See https://github.com/Microsoft/TypeScript-wiki/blob/main/Breaking-Changes.md#extending-built-ins-like-error-array-and-map-may-no-longer-work where it first appeared. 

I'm unsure if this is only an issue for certain compilation targets or not, however I do have a memory from the past where not having this line has caused problems

Reply:
Reading around a bit I believe this is specifically for ES5 targets, see also https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error

Reply:
Ah huh! Okay... I guess one more naive question, but this makes it sound like the es2015 target doesn't have this issue:

https://www.dannyguo.com/blog/how-to-fix-instanceof-not-working-for-custom-errors-in-typescript/#upgrade-the-compilation-target

And given it's ~7 years after 2015 :-D do you think we could just drop this as probably not necessary anymore? Like is this specifically required for your app/target's target environment?

Reply:
me personally nope, I'm targeting es2015 for my applications. I'll check what is the default support for things like create-react-app and the angular cli. If they support es5 out of the box as default I think grpc-web probably should too?

Reply:
ok yep I create a fresh react app with typescript and tried it out and the instanceof check works fine. Happy to drop it

Reply:
removed.